src/common/Types.ts

```diff
@@ -976,6 +976,8 @@ export interface NavigatorProps extends CommonProps {
     transitionCompleted?: () => void;
     cardStyle?: ViewStyleRuleSet;
     children?: ReactNode;
+    initialRouteStack?: Array<NavigatorRoute>;
```

Comment:
Why do you want to pass the initial route stack as props? In general, I'm not a fan of props that specify an "initial" value because they go against the React philosophy of updating existing components. It's not clear, for example, what happens if the caller changes the initialRouteStack and initialRoute on subsequent renders. Should they be honored or not?

If we do keep these, they need to apply to both the native and web implementations. It looks like you've implemented it only on the web currently.